app/models/user.rb

```diff
@@ -3,6 +3,9 @@ class User < ApplicationRecord
   has_many :comments
   has_many :likes
 
+  validates :name, presence: true
+  validates :posts_counter, numericality: { greater_than_or_equal_to: 0 }
```

Comment:
- [ ] I think it would be great if you add validation to `Only integre`  since it is mentioned in the requirements so I recommend adding it you can check this link so you can add like this [ruby validation](https://guides.rubyonrails.org/active_record_validations.html) ✅💯

![image](https://github.com/Lawmsangi/blog-app/assets/58553711/3da38f20-0ef5-40a9-8ce5-0a349caf2702)